src/experiments/routing/simple_graph.rs

```diff
@@ -118,14 +137,41 @@ mod tests {
 
     fn get_test_graph() -> Graph {
         let vertex_list = vec![Vertex { id: 0 }, Vertex { id: 1 }, Vertex { id: 2 }];
-        let edge_list = vec![Edge { id: 1, from: 0, to: 1, weight: 1.0 },
-                             Edge { id: 2, from: 0, to: 2, weight: 2.0 },
-                             Edge { id: 3, from: 1, to: 1, weight: 1.0 },
-                             Edge { id: 4, from: 1, to: 2, weight: 4.0 },
-                             Edge { id: 5, from: 2, to: 0, weight: 2.0 }];
+        let edge_list = vec![
```

Comment:
In case you want to preserve the previous formatting due to readability, it is also possible to use `#[rustfmt::skip]` to prevent formatting. I have no oppinion on what is better here.

Reply:
Thank's for the hint! Since those are just experiments, which will probably be deleted anyway, it's no problem.